modules/apps/foundation/contacts/contacts-web/src/main/java/com/liferay/contacts/web/notifications/ContactsCenterUserNotificationHandler.java

```diff
@@ -108,6 +102,16 @@ protected String getBody(
 				new Object[] {creatorUserName});
 		}
 
+		if ((socialRequest.getStatus() !=
+				SocialRequestConstants.STATUS_PENDING) ||
+			(socialRequest.getCreateDate() >
+				userNotificationEvent.getTimestamp())) {
+
+			return StringUtil.replace(
+				_BODY_TEMPLATE_DEFAULT, new String[] {"[$BODY$]", "[$TITLE$]"},
```

Comment:
You're right. I agree there should be a default title for other types of requests. I'll add it and update the pull.